.circleci/sync-service.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: testground-sync-service
+spec:
+  selector:
+    matchLabels:
+      name: testground-sync-service
+  template:
+    metadata:
+      labels:
+        name: testground-sync-service
+    spec:
+      terminationGracePeriodSeconds: 10
+      hostPID: true
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+        - name: testground-sync-service
+          image: iptestground/sync-service:latest
+          command: ["/service"]
+          env:
+            - name: REDIS_HOST
+              value: "172.18.0.1"
```

Comment:
The network created for "kind" is 172.18.0.0/16, where the host of the containers is 172.18.0.1. We need to be able to access Redis which is running on a Docker container on host from the pod where the sync service is running.